shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -1053,6 +1053,18 @@ private void diposeAllViews() {
     }
   }
 
+  public void onTrimMemory(int level) {
+    for (VirtualDisplayController vdc : vdControllers.values()) {
+      vdc.clearSurface();
+    }
+  }
+
+  public void onResume() {
+    for (VirtualDisplayController vdc : vdControllers.values()) {
+      vdc.resetSurface();
```

Comment:
Did you want to only do this on api 14?

Reply:
Apparently we are better Android citizens if we release these resources so I think we want it to run on all versions.